pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -1,136 +1,228 @@
-                });
-            </script>
+          </div>
+        </div>
+
+        <table id="questionsTable"
+               data-data="<%= JSON.stringify(questions) %>"
+               data-classes="table table-sm table-hover table-bordered"
+               data-thead-classes="thead-light"
+               data-filter-control="true"
+               data-show-filter-control-switch="true"
+               data-show-search-clear-button="true"
+               data-show-columns="true"
+               data-show-columns-toggle-all="true"
+               data-show-button-text="true"
+               data-pagination="true"
+               data-pagination-v-align="top"
+               data-page-list="[50,100,200,500,unlimited]"
+               data-page-size="50"
+               data-smart-display="false"
+               data-show-extended-pagination="true"
+               data-toolbar=".fixed-table-pagination"
+               data-sticky-header="true">
+          <thead>
+            <tr>
+              <th data-field="qid"
+                  data-sortable="true"
+                  data-class="align-middle sticky-column"
+                  data-formatter="qidFormatter"
+                  data-filter-control="input"
+                  data-switchable="true">QID</th>
+              <th data-field="title"
+                  data-sortable="true"
+                  data-class="align-middle text-nowrap"
+                  data-filter-control="input"
+                  data-filter-custom-search="genericFilterSearch"
+                  data-switchable="true">Title</th>
+              <th data-field="topic"
+                  data-sortable="true"
+                  data-class="align-middle text-nowrap"
+                  data-formatter="topicFormatter"
+                  data-filter-control="select"
+                  data-filter-data="func:topicList"
+                  data-filter-custom-search="badgeFilterSearch"
+                  data-switchable="true">Topic</th>
+              <th data-field="tags"
+                  data-sortable="false"
+                  data-class="align-middle text-nowrap"
+                  data-formatter="tagsFormatter"
+                  data-filter-control="select"
+                  data-filter-data="func:tagsList"
+                  data-filter-custom-search="badgeFilterSearch"
+                  data-switchable="true">Tags</th>
+              <th data-field="display_type"
+                  data-sortable="true"
+                  data-class="align-middle text-nowrap"
+                  data-formatter="versionFormatter"
+                  data-filter-control="select"
+                  data-filter-data="func:versionList"
+                  data-filter-custom-search="badgeFilterSearch"
+                  data-switchable="true">Version</th>
+              <% locals.course_instances.forEach(course_instance => { %>
+              <th data-field="assessments_<%= course_instance.id %>"
+                  data-sortable="true"
```

Comment:
This column has sort buttons but they don't seem to do anything when I click them.

Reply:
Sort by assessment removed.